lib/src/widgets/scroll_to_bottom.dart

```diff
@@ -3,31 +3,40 @@ part of dash_chat;
 class ScrollToBottom extends StatelessWidget {
   final Function onScrollToBottomPress;
   final ScrollController scrollController;
+  final double bottomPosition;
+  final Color backgroundColor;
+  final Color textColor;
 
   ScrollToBottom({
     this.onScrollToBottomPress,
     this.scrollController,
+    this.bottomPosition,
+    this.backgroundColor,
+    this.textColor,
   });
 
   @override
   Widget build(BuildContext context) {
     return Container(
-      width: 48.0,
-      height: 48.0,
+      width: 35,
```

Comment:
Also, if we are providing `scrollToBottom` Button properties we can have dimensions property for setting the height & width of the button.

Reply:
Yes, we can add `height` and `width` to the map I mentioned below